plugins/auth-backend/src/providers/cloudflare-access/provider.ts

```diff
@@ -269,20 +280,31 @@ export class CloudflareAccessAuthProvider implements AuthProviderRouteHandlers {
       };
     }
     const claims = verifyResult.payload as CloudflareAccessClaims;
+
     // Builds a passport profile from JWT claims first
     try {
-      // If we successfully fetch the get-identity endpoint,
-      // We supplement the passport profile with richer user identity
-      // information here.
-      const cfIdentity = await this.getIdentityProfile(jwt);
+      let cfIdentity: CloudflareAccessIdentityProfile;
+      if (isServiceToken) {
+        cfIdentity = {
+          id: subject,
+          name: 'Bot',
+          email: `${subject}@${this.teamName}.com`,
```

Comment:
Is this a well known way of forming that email or is it just to be seen as a dummy value? The domain part of that mail is likely to be wrong for a lot of orgs.

Reply:
Excellent point. I don't think there's a way around putting this in app-config. I'll update it.